modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js

```diff
@@ -56,16 +56,22 @@ export default function SearchBar({
 	const alignElementRef = useRef();
 	const dropdownRef = useRef();
 
-	const characterThresholds = JSON.parse(suggestionsContributorConfiguration)
-		.filter((config) => config.attributes?.characterThreshold)
-		.map((config) => parseInt(config.attributes.characterThreshold, 10));
-
 	// If the blueprint suggestions contributor does not set its own threshold,
 	// the suggestions display threshold will default to the global one.
 
-	const finalSuggestionsDisplayThreshold = characterThresholds.length
-		? Math.min(...characterThresholds)
-		: parseInt(suggestionsDisplayThreshold, 10);
+	const _getLowestSuggestionsDisplayThreshold = useCallback(() => {
+		const characterThresholds = JSON.parse(
```

Comment:
To make it easier to distinguish variable type when arrays are involved, let's try to suffix the variable with `*Array`.